app/javascript/components/CollectionDetails.js

```diff
@@ -67,7 +67,7 @@ const CollectionDetails = ({ content = '', email = '', website = '' }) => {
         { website &&
           <>
             <dt style={descriptionStyle}>Website:</dt>
-            <dd><a href={website}>{website}</a></dd>
+            <dd dangerouslySetInnerHTML={{ __html: website }}></dd>
```

Comment:
@phuongdh @Dananji This is equivalent to `innerHTML` and while it is called `dangerously` because of the possibility of XSS I think it should be safe here since we're in control of the html.  I thought it made more sense for the presenter to return a full a tag instead of the label and url separately.  What do you two think?

Reply:
Yeah, sounds good to me :+1: